### DISCUSSION TITLE
Trying to set offsets at terminal with GCODE

### DISCUSSION THREAD

**Discussion by tekjock** [mriscoc/Ender3V2S1#540] [Q&A]
HI, 

Let me start by saying this firmware is amazing. With that out of the way!

I am on day one of useing this firmware and I am in ocotprint on the terminal tab and trying to set my offsets with this
```
G0 X1.5 Y6 Z0 ; set home position
M428 ; Set offset
m500 ; Store settings
```
This worked with the stock ender firmware, but the M428 command is not working to set the offset. 

I have tried it on the screen, but, I home it, move the Y axis 6, and store the settings, but nothing saves

is there a alternative to M428, or what are the exact steps with this firmware to set the offsets.  

when printing something that goes to the  full edge, it prints right on the very edge. Before i set Y6 and it was perfect.

thanks in advanced.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by tekjock** [reply]
Thank you!
